templates/details/overview.html

```diff
@@ -13,6 +13,29 @@ <h4 class="p-heading--5 u-no-margin--bottom">About</h4>
         </p>
         <hr class="p-separator--shallow">
       {% endif %}
+      {% if package.type == "bundle" %}
+        {% if not package.result.summary %}
+          <h4 class="p-heading--5 u-sv1 u-no-margin--bottom">About</h4>
+        {% endif %}
+          <div class="p-topology-info" data-js="topology-info">
+            <div class="p-card--highlighted">
+              <object data="https://api.jujucharms.com/charmstore/v5/bundle/{{ package.name }}/diagram.svg" type="image/svg+xml" style="pointer-events: none;"></object>
+              <i class="p-icon--fullscreen" role="button" tabIndex="0">Show fullscreen</i>
+            </div>
+          </div>
+          <div class="p-modal u-hide" data-js="topology-modal">
+            <div class="p-modal__dialog" role="dialog" aria-labelledby="modal-title" aria-describedby="modal-description">
+              <header class="p-modal__header">
+                <h2 class="p-modal__title" id="modal-title">{{ package.name }}</h2>
+                <button class="p-modal__close" aria-label="Close active modal" aria-controls="modal">Close</button>
```

Comment:
label could be more descriptive. For example, "Close the topology modal"

Reply:
There is no `id="model"` which is declared in the `aria-controls` value. I think you mean to add the id to the model container?